common/scala/src/main/scala/whisk/core/mesos/MesosContainerFactory.scala

```diff
@@ -66,7 +67,13 @@ case class MesosConfig(masterUrl: String,
                        constraints: Seq[String],
                        constraintDelimiter: String,
                        blackboxConstraints: Seq[String],
-                       teardownOnExit: Boolean) {}
+                       teardownOnExit: Boolean,
+                       healthCheck: Option[HealthCheckConfig],
+                       offerRefuseDuration: FiniteDuration,
+                       taskLaunchTimeout: FiniteDuration,
+                       taskDeleteTimeout: FiniteDuration,
+                       subscribeTimeout: FiniteDuration,
+                       teardownTimeout: FiniteDuration) {}
```

Comment:
Have you considered putting this into an encapsulating type, like `TimeoutConfig`?